### PR TITLE
Add repository double-conversion support for loongarch

### PR DESCRIPTION
**Pull request by nmanhong**
Add warehouse double-conversion to support loongarch, which has been tested on loongarch and can be passed

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -9,6 +9,7 @@
 [submodule "third_party/double-conversion"]
 	path = third_party/double-conversion
 	url = https://github.com/google/double-conversion
+	branch = master
```
Comment (mthrok) [inline .gitmodules:12]: Does this mean each `git submodule` will fetch the latest commit from `master` branch?
I think it is preferable to pin the commit to the current latest in this case.

**Review by mthrok — COMMENTED** [review]
@Nayef211 @malfet Are you familiar with loongarch?
I guess this PR is safe as long as the basic double-conversion functionalities are tested on upstream.

File: .gitmodules (hunk 1 above)
Comment (nmanhong) [reply .gitmodules:12]: According to my understanding, it should not be pulled from the latest every time, but when adding submodules, a branch is specified.  it should be from the commit 256ac80956 of this branch at that time. Manually update this submodule. Maybe I don't know much about github.

Comment (joecummings) [reply .gitmodules:12]: I think this should be `main` branch

Comment (Nayef211) [reply .gitmodules:12]: @nmanhong I think if you want to update the submodule, you can go into the directory and run `git fetch` and `git merge` the upstream branch to update the local code. In this case the upstream branch would be the `master` branch. AFAIK this doesn't need to be specified in the `.gitmodules` file.

**Comment by Nayef211** [comment]
> @Nayef211 @malfet Are you familiar with loongarch? I guess this PR is safe as long as the basic double-conversion functionalities are tested on upstream.

I'm not, but the changes aren't breaking any new CI tests. I'm okay with merging it unless you think we should get eyes from someone that's more familiar with `loongarch`

File: .gitmodules (hunk 1 above)
Comment (Nayef211) [reply .gitmodules:12]: @nmanhong I'm happy to merge this once we've removed the following line from this PR.

**Pull request closed without merging**